utils/fixed_decimal/src/decimal.rs

```diff
@@ -989,6 +1006,35 @@ impl FixedDecimal {
     /// assert_eq!("1", dec.to_string());
     /// ```
     pub fn trunc(&mut self, position: i16) {
+        self.trunc_with_increment(position, RoundingIncrement::I1)
+    }
+
+    /// Truncates the number on the right to a particular position and rounding increment,
+    /// deleting digits if necessary.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use fixed_decimal::{FixedDecimal, RoundingIncrement};
+    /// # use std::str::FromStr;
+    ///
+    /// let mut dec = FixedDecimal::from_str("-3.5").unwrap();
+    /// dec.trunc_with_increment(0, RoundingIncrement::I2);
+    /// assert_eq!("-2", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("7.57").unwrap();
+    /// dec.trunc_with_increment(-1, RoundingIncrement::I5);
+    /// assert_eq!("7.5", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("5.45").unwrap();
+    /// dec.trunc_with_increment(-2, RoundingIncrement::I25);
+    /// assert_eq!("5.25", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("9.99").unwrap();
+    /// dec.trunc_with_increment(-1, RoundingIncrement::I25);
+    /// assert_eq!("7.5", dec.to_string());
+    /// let mut dec = FixedDecimal::from_str("9.99").unwrap();
+    /// dec.trunc_with_increment(-2, RoundingIncrement::I2);
+    /// assert_eq!("9.98", dec.to_string());
+    /// ```
+    pub fn trunc_with_increment(&mut self, position: i16, increment: RoundingIncrement) {
```

Comment:
naming: I'm not convinced `with_increment` is the best name for this operation. Maybe something like `trunc_to_precision` or `trunc_coarse`?